tools/unitctl/unitctl/src/cmd/instances.rs

```diff
@@ -51,7 +51,7 @@ pub(crate) async fn cmd(args: InstanceArgs) -> Result<(), UnitctlError> {
                         if !sock_path.is_dir() {
                           return Err(UnitctlError::ControlSocketError{
                                 kind: ControlSocketErrorKind::General,
-                                message: format!("user must specify a directory of UNIX socket directory"),
+                                message: "user must specify a directory of UNIX socket directory".to_string(),
```

Comment:
this is something RustRover picked up. I don't feel strongly about it, being new to the language, but the suggestion made sense ("don't use format without placeholders and arguments")